drizzle-orm/src/monomigrator.ts

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable import/extensions */
+import type { AwsDataApiPgDatabase } from './aws-data-api/pg/index.ts';
+import type { BetterSQLite3Database } from './better-sqlite3/index.ts';
+import type { BunSQLiteDatabase } from './bun-sqlite/index.ts';
+import type { DrizzleD1Database } from './d1/index.ts';
+import { entityKind } from './entity.ts';
+import type { LibSQLDatabase } from './libsql/index.ts';
+import type { MigrationConfig } from './migrator.ts';
+import type { MySql2Database } from './mysql2/index.ts';
+import type { NeonHttpDatabase } from './neon-http/index.ts';
+import type { NeonDatabase } from './neon-serverless/index.ts';
+import type { NodePgDatabase } from './node-postgres/index.ts';
+import type { PlanetScaleDatabase } from './planetscale-serverless/index.ts';
+import type { PostgresJsDatabase } from './postgres-js/index.ts';
+import type { TiDBServerlessDatabase } from './tidb-serverless/index.ts';
+import type { VercelPgDatabase } from './vercel-postgres/index.ts';
+
+export async function migrate(
```

Comment:
could this be the source of that migrate error you were seeing @Rodriguespn ? idk

Reply:
It was a dependency issue. Once I reintalled the dependencies it worked

Reply:
ah nice